service/history/replicationTaskProcessor.go

```diff
@@ -210,11 +210,9 @@ func (p *ReplicationTaskProcessorImpl) eventLoop() {
 			))
 
 		case <-cleanupTimer.C:
-			if p.config.EnableCleanupReplicationTask() {
-				if err := p.cleanupReplicationTasks(); err != nil {
-					p.logger.Error("Failed to clean up replication messages.", tag.Error(err))
-					p.metricsClient.Scope(metrics.ReplicationTaskCleanupScope).IncCounter(metrics.ReplicationTaskCleanupFailure)
-				}
+			if err := p.cleanupReplicationTasks(); err != nil {
```

Comment:
Looks like this was previously guarded by `EnableCleanupReplicationTask` flag.  Now looks like this is always needed.  Just want to confirm the intention is if `GlobalNamespace` is enabled then we want run cleanupReplicationTasks in all cases?

Reply:
replicationTaskProcessor will only be initialize when there are other clusters to poll from

Reply:
Might be worth mentioning this as a comment here.